src/mame/machine/esqpanel.h

```diff
@@ -107,6 +112,24 @@ class esqpanel_device : public device_t, public device_serial_interface
 
 	esqpanel::external_panel_server *m_external_panel_server;
 
+	const std::string slashes_to_separators(const std::string s) const {
+		std::string t(s);
+		size_t start_pos = 0;
+		while (start_pos != std::string::npos) {
+			size_t slash = t.find('/', start_pos);
+			if (slash == std::string::npos) {
+				start_pos = slash;
+			} else {
+				t.replace(slash, 1, PATH_SEPARATOR);
+				start_pos = slash + 1;
+			}
+		}
+#if DEBUG_ESQPANEL
+		std::cerr << "slashes_to_separators: '" << s << "' -> '" << t << "'" << std::endl;
```

Comment:
Use `logerror` and the mechanisms using `logmacro.h` for debug logging, not stuff like this.

Reply:
Done.